CFC_WebApp/main/mode/util.py

```diff
@@ -0,0 +1,255 @@
+
+from pykml import parser
+from pykml.parser import Schema
+from pykml.factory import KML_ElementMaker as KML
+from os import path
+import json
+import geojson
+from lxml import etree
+import sys, os, random
+from datetime import datetime
+from random import randrange
+
+
+sampleTrip = '{"date": "20140413", "lastUpdate": "20140414T064442Z", "segments": [], "summary": null}'
+sampleSegment = '{"place": {"type": "unknown", "id": 54095848, "location": {"lat": 37.3910149202, "lon": -122.0865010796}}, "endTime": "20140413T234434-0700", "type": "place", "startTime": "20140412T190446-0700", "lastUpdate": "20140414T064442Z"}'
+
+def get_clusters_info():
```

Comment:
This is dead code and can be removed since it has been moved into the recommender pipeline.